### ISSUE TITLE
Dataset for grok

### ISSUE THREAD

**Issue by Konard** [xai-org/grok-1#61]
That is the dataset for grok?
Will it be open-sourced?
How to use that dataset to retrain the model?

**Comment by Whytehorse2022** [comment]
I'm also interested in a description of the training data.

**Comment by grothedev** [comment]
yes. if the training data isn't released (at least the titles of the data items) then it's not technically completely open source.
